gamemodes/horde/gamemode/sh_misc.lua

```diff
@@ -66,3 +66,18 @@ net.Receive("Horde_Void_Shield_Remove", function ()
 end)
 end
 -- Draw the antimatter shield
+
+function HORDE.traceSolidIgnoreAllies(ply)
```

Comment:
Can you make this `HORDE:traceSolidIgnoreAllies` all horde functions are defined as methods for whatever reason, it'd be best to keep consistency